practic_module_3/task4.cpp

```diff
@@ -0,0 +1,16 @@
+﻿#include <iostream>
+
+int main()
+{
+	// task 4:----------------------------------------------------------------------------------
+	// Write a program that displays your favorite poem.Use an appropriate escape sequence
+	// for the line breaks.If you don’t have a favorite poem, you can borrow this one by Ogden
+	// Nash:
+	//			Candy is dandy,
+	//			But liquor is quicker.
+
+
+	std::cout << '\t' << "Candy is dandy, " << "\n" << '\t' << "But liquor is quicker." << std::endl << std::endl;
+
+	return 0;
```

Comment:
NIT: read about EXIT_SUCCESS and EXIT_FAILURE constants
https://en.cppreference.com/w/c/program/EXIT_status